certsuite/harness.py

```diff
@@ -382,19 +382,19 @@ def list_tests(args, config):
         print "%s:%s" % (test, group)
     return True
 
-def edit_device_profile(device_profile_path):
+def edit_device_profile(device_profile_path, question):
     resp = ''
+    result = None
     try:
-        env = environment.get(environment.InProcessTestEnvironment, addr=None, verbose=False)
+        env = environment.get(environment.InProcessTestEnvironment)
         url = "http://%s:%d/profile.html" % (env.server.addr[0], env.server.addr[1])
         webbrowser.open(url)
         environment.env.handler = server.wait_for_client()
-        
-        question = ''
+
         resp = environment.env.handler.prompt(question)
 
-        message = 'Create device profile failed!!'    
-        if resp != '':
+        message = 'Create device profile failed!!'
+        if resp != '' and resp != None:
```

Comment:
for the better way to use python, you should use
```python
if resp is not None and resp != '':
```
It brings better performance and easier for process.